### ISSUE TITLE
Edge ports can cause dot to crash

### ISSUE THREAD

**Issue by thomasdullien** [ellson/graphviz#1231]
Hey there,

(Source line numbers refer to GV 2.36.0)

The attached example .dot file crashes dot in dotsplines.c, line 1301 - ED_label(auxe)
ends up being 0, and ED_label(auxe)->pos predictably segfaults.

According to valgrind, an invalid write8 happens previously, in line 1231. I am unfortunately
too unfamiliar with all the macros in the codebase to judge exactly what is going on, but it
appears that cloneEdge does not work as expected. There are a few lines of code commented
out w/o further details - perhaps they are the culprit?

Cheers,
Thomas
[corrupt_tree_part.dot.txt](https://github.com/ellson/graphviz/files/943546/corrupt_tree_part.dot.txt)

**Comment by thomasdullien** [comment]
Thanks for the quick response :-), and apologies for the false bug report. I had git-cloned the master repository and ran ./autogen.sh followed by Make, and then used the dot executable ./cmd/dot/.libs/dot instead of the ./cmd/dot/dot executable. Would it make sense to remove the 2.36.0 executable in ./cmd/dot/.libs/dot to avoid confused bug reports like mine? :)

**Comment by ellson** [comment]
I don't think you got that executable from the git repo. Perhaps trying running "make distclean" before updating and rebuilding?

But, as you mention, using ./cmd/dot/dot should be immune to the clutter of a previous build/
